linux_os/guide/services/xwindows/disabling_xwindows/xwindows_runlevel_setting.rule

```diff
@@ -4,6 +4,13 @@ prodtype: rhel7,fedora
 
 title: 'Disable X Windows Startup By Setting Default Target'
```

Comment:
Prefer that xwindows_runlevel_setting.rule be a different rule for rhel7. It should be xwindows_runlevel_target.rule

Reply:
@redhatrises ok, done.